aiohttp_pydantic/oas/docstring_parser.py

```diff
@@ -120,6 +120,21 @@ def tags(docstring: str) -> List[str]:
     return []
 
 
+def security(docstring: str) -> List[dict] | None:
```

Comment:
We must use Union or Optional instead of `|` to keep compatibility with python 3.8 and 3.9

```
def security(docstring: str) -> Optional[List[dict]]:
```

Reply:
Optional is now used to keep compatibility